storage/engine/rocksdb_test.go

```diff
@@ -297,6 +296,52 @@ func BenchmarkMVCCScan100Versions1000Rows(b *testing.B) {
 	runMVCCScan(1000, 100, b)
 }
 
+// runMVCCGet first creates test data (and resets the benchmarking
+// timer). It then performs b.N MVCCGets.
+func runMVCCGet(numVersions int, b *testing.B) {
+	// Use the same number of keys for all of the mvcc get
+	// benchmarks. Using a different number of keys per test gives
+	// preferential treatment to tests with fewer keys. Note that the
+	// datasets all fit in cache and the cache is pre-warmed.
+	const numKeys = 100000
+
+	rocksdb := setupMVCCScanData(numVersions, numKeys, b)
+	defer rocksdb.Stop()
+
+	prewarmCache(rocksdb)
+
+	b.SetBytes(1024)
+	b.ResetTimer()
+
+	b.RunParallel(func(pb *testing.PB) {
+		for pb.Next() {
+			// Choose a random key to retrieve.
+			keyIdx := rand.Int31n(int32(numKeys))
+			key := proto.Key(encoding.EncodeVarUint32([]byte("key-"), uint32(keyIdx)))
+			walltime := int64(5 * (rand.Int31n(int32(numVersions)) + 1))
+			ts := makeTS(walltime, 0)
+			_, err := MVCCGet(rocksdb, key, ts, nil)
+			if err != nil {
```

Comment:
Not like it matters but unless you need the results, better to combine these two statements into a single if.